src/Framework/AssemblyInfo.cs

```diff
@@ -43,6 +43,7 @@
 
 #if STATIC_VERSION_NUMBER
 [assembly: AssemblyVersion("15.1.0.0")]
```

Comment:
Tried that and reverted the change because most of the unit test dlls stopped compiling with conflicting types. This is because MSBuild is using shared sources instead of project dependencies. Once we replace all the shared sources with p2p dependencies, we can also replace this string with a constant :)